topics/sentinel.md

```diff
@@ -785,12 +795,33 @@ configured with `requirepass`, the Sentinel configuration must include the
 Configuring Sentinel instances with authentication
 ---
 
-You can also configure the Sentinel instance itself in order to require
-client authentication via the `AUTH` command, however this feature is
-only available starting with Redis 5.0.1.
+Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. Starting with Redis 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis 5.0.1) support password-only authentication. 
+
+Note that Sentinel's authentication configuration should be **applied to each of the instances** in your deployment, and **all instances should use the same configuration**. Furthermore, ACL and password-only authentication should not be used together.
+
+### Access Control List authentication
+
+The first step in securing a Sentinel instance with ACL is preventing any unauthorized access. To do that, you'll need to disable the default superuser (or at the very least set it up with a strong password) and create a new one:
+
+    127.0.0.1:5000> ACL SETUSER admin ON >adminpassword +@all
+    OK
+    127.0.0.1:5000> ACL SETUSER default off
+    OK
+
+The default user is used by Sentinel to connect to other instances. You can provide the credentials of another superuser with the following configuration directives:
+
+    sentinel sentinel-user <username>
+    sentinel sentinel-pass <password>
+
+Where `<username>` and `<password>` are the Sentinel's ACL user and password, respectively.
+
+Lastly, for authenticating incoming client connections, you can create a restricted Sentinel user like this:
+
+    127.0.0.1:5000> ACL SETUSER sentinel-user ON >user-password -@all +@connection +SENTINEL|MASTERS +SENTINEL|MASTER +SENTINEL|GET-MASTER-ADDR-BY-NAME
```

Comment:
can someone confirm that this line would be enough for sentinels to talk to each other and to their job?

Reply:
This is actually for incoming client connections and should be enough for discoverability, but an affirmation would be good.